### DISCUSSION TITLE
No WOTC Button?

### DISCUSSION THREAD

**Discussion by ProjectKaizen** [X2CommunityCore/xcom2-launcher#276] [Help] [answered]
I downloaded the latest version of the AML and after extracting, following instructions, and adding the steam Goldberg patch, it worked except the only button present within is a launcher for the Vanilla. Any fix or recommendations I should follow?

**Comment by RevZero** [comment, marked as answer]
Make sure you base path looks something like this:
https://github.com/X2CommunityCore/xcom2-launcher/wiki/Getting-started#settings

Do not point it to the WotC sub-folder.

**Reply by ProjectKaizen** [reply]
Thanks for the tip, and after a thorough re-evaluation of the proper setup, my smooth brain missed editing the game directory field.
